README.md

```diff
@@ -4,9 +4,13 @@ Full stack coder...  Actually more like a jack of all trades, ever adapting (rei
 Deep INITIAL impression of COMPUTERS from TRS80 days in 1980 at Chapel School's math club...  WOW!...
 COBOL coding (punching cards stacks being dropped accidentally while on way to SYSOP terminal room at FLHS)...
 VIC-20 5K days...  5K!!!  That's a LOT of memory!
-AP COMPUTER class PIONEER...  We actually knwe more than the instructor...
+AP COMPUTER class PIONEER...  We actually know more than the instructor...
 PC compatibles...  Life is but a dream...  We can do anything!
 RS232 handshaking...  So cool...  We can digitize our world, just add imagination...
 Game coding...  Arcane disciplined art of digital DM
-App Programming and Solution...  Cheating to pay the inevitable bills...
+Applications Programming and "Solutions"...  Cheating to pay the inevitable bills...
 Selling ideas to teams, customers and other sellers...  A prelude to API age...
+IoT, Big Data, Data Visualization, ML, AI, and...  Python!...  NEW jargons, old familiar faces...  Back 2 Future XXIII, a Disney prjt...
+Unicorn or bust...  The age of modern robber barrons...
+COVID 19 >> levels the playing field >> we all humans <SURPRISE>
+OK boomer...
```

Comment:
Added lines to reflect CURRENT thinking process and target areas...

Reply:
CURRENT as in "as of April 5th, 2020..."

Reply:
thinking process... not necessarily the writer's but more of a subjective observation/consensus of   collective trends in the perspective of the writer...

Reply:
target areas...  writer's own personal formulaic direction based on perceived drivers...

Reply:
COVID 19 likely an inflection point for years to come...